ftplugin/php/commands.vim

```diff
@@ -1,130 +1,101 @@
-""
-" Update Phpactor to the latest version using the branch
-" defined with @setting(g:phpactorBranch)
-command! -nargs=0 PhpactorUpdate call phpactor#Update()
-
-""
-" Clear the entire cache - this will take effect for all projects.
-command! -nargs=0 PhpactorCacheClear call phpactor#CacheClear()
-
-""
-" Show some information about Phpactor's status
-command! -nargs=0 PhpactorStatus call phpactor#Status()
-
-""
-" Dump Phpactor's configuration
-command! -nargs=0 PhpactorConfig call phpactor#Config()
-
 ""
 " Extract a new method from the current selection
-command! -range=% PhpactorExtractMethod call phpactor#ExtractMethod()
+command! -buffer -range=% PhpactorExtractMethod call phpactor#ExtractMethod()
 
 ""
 " Extract the selected expression and assign it to a variable before (placing
 " it before the current statement)
-command! -range=% PhpactorExtractExpression call phpactor#ExtractExpression('v')
+command! -buffer -range=% PhpactorExtractExpression call phpactor#ExtractExpression('v')
 
 ""
 " Extract a constant from a literal
-command! -nargs=0 PhpactorExtractConstant call phpactor#ExtractConstant()
+command! -buffer -nargs=0 PhpactorExtractConstant call phpactor#ExtractConstant()
 
 ""
 " Expand the class name under the cursor to it's fully-qualified-name
-command! -nargs=0 PhpactorClassExpand call phpactor#ClassExpand()
+command! -buffer -nargs=0 PhpactorClassExpand call phpactor#ClassExpand()
```

Comment:
ClassExpand is useful on XML and YAML files (or any text document)

Reply:
Moved